lib/watir/elements/table.rb

```diff
@@ -69,7 +69,7 @@ def cell_size_check(header_row, cell_row)
 
       index = cell_row.selector[:index]
       row_id = index ? "row at index #{index - 1}" : 'designated row'
-      raise Exception::Error, "#{row_id} has #{row_size} cells, while header row has #{header_size}"
+      raise WatirException, "#{row_id} has #{row_size} cells, while header row has #{header_size}"
```

Comment:
The people can be rescuing those, what's the need for the change?

Reply:
yeah, you are right. I thought it read a little better in the code, but when it gets thrown, the whole namespacing will be there, so not really an improvement. Changed it back.